pennylane/devices/default_qubit_autograd.py

```diff
@@ -158,3 +160,17 @@ def _get_unitary_matrix(self, unitary):
             return unitary.eigvals
 
         return unitary.matrix
+
+    def map_wires(self, wires):
+        """Map the wire labels of wires using this device's wire map.
+
+        Args:
+            wires (Wires): wires whose labels we want to map to the device's internal labelling scheme
+
+        Returns:
+            Wires: wires with new labels
+        """
+        wires = Wires([w.item() if isinstance(w, tensor) else w for w in wires])
+        mapped_wires = super().map_wires(wires)
+
+        return mapped_wires

```

Comment:
Since this is the same code with `default_qubit_tf`, would it make sense to move this code to `DeviceQubit`?

Reply:
This would make all `QubitDevice`s, including the default qubits and default mixed state qubit, check for tensor instances, which maybe is not necessary?